robotidy/disablers.py

```diff
@@ -19,6 +19,18 @@ def wrapper(self, node, *args, **kwargs):
     return wrapper
 
 
+def get_section_name_from_header_type(node):
+    header_type = node.header.type if node.header else "COMMENT HEADER"
+    return {
+        "SETTING HEADER": "settings",
+        "VARIABLE HEADER": "variables",
+        "TESTCASE HEADER": "tests",
```

Comment:
I'm just thinking if it should not be named `testcases`, at the section name...
`tests` is fine I guess, but I wanted to comment on that.

Reply:
My heart prefers tests but my brain tells me it should be testcases because it will follow convention (name from the section name). And my guts decoded to combine both approach and both testcases and tests will work - since we're the ones that map it to settings we can allow both versions (but I will mention only testcases in documentation).